app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_locale
-
+  include SessionsHelper
```

Comment:
file session_helper dùng để viết các hàm không liên quan trực tiếp đến logic lẫn flow của bài toán. trong trường hợp này thì viết các hàm liên quan đến session: login, logout, check login